dev/tests/js/jasmine/tests/app/code/Magento/Ui/base/js/grid/data-storage.test.js

```diff
@@ -22,7 +25,253 @@ define([
             });
         });
 
-        describe('hasScopeChanged', function () {
+        describe('"initConfig" method', function () {
+
+            var model = new DataStorage({
+                dataScope: 'magento'
+            });
+
+            it('Check for defined ', function () {
+                expect(model.hasOwnProperty('initConfig')).toBeDefined();
+            });
+
+            it('Check method type', function () {
+                var type = typeof model.initConfig;
+
+                expect(type).toEqual('function');
+            });
+
+            it('Check returned value if method called without arguments', function () {
+                expect(model.initConfig()).toBeDefined();
+            });
+
+            it('Check returned value type if method called without arguments', function () {
+                var type = typeof model.initConfig();
+
+                expect(type).toEqual('object');
+            });
+
+            it('Check this.dataScope property (is modify in initConfig method)', function () {
+                model.dataScope = null;
+                model.initConfig();
+                expect(typeof model.dataScope).toEqual('object');
+            });
+
+            it('Check this._requests property (is modify in initConfig method)', function () {
+                model._requests = null;
+                model.initConfig();
+                expect(typeof model._requests).toEqual('object');
+            });
+        });
+
+        describe('"getByIds" method', function () {
+
+            var model = new DataStorage({
+                dataScope: 'magento'
+            });
+
+            it('check for defined', function () {
```

Comment:
Do you think `check for defined` and `check method type` are necessary? If the function is not declared the following tests will fail anyway and it wouldn't be hard to identify the problem.